x-pack/plugins/files/server/blob_storage_service/adapters/es/mappings.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MappingTypeMapping } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+
+export const mappings: MappingTypeMapping = {
+  properties: {
+    chunk: { type: 'long' },
+    content: { type: 'object', enabled: false }, // Base64 encoded content
+    size: { type: 'long' },
```

Comment:
Can we make this mapping such that it has a single "binary" field?

```suggestion
    data: { type: 'binary' },
```

Or do we need the `chunk` and `size` fields for something?

Reply:
This idea was cherry picked from how Reporting is currently chunking stored reports in ES. The `chunk` and `size` values are used when it comes time to download.